MetaMask/metamask-extension
# fix: BlockaidBannerAlert LoadingIndicator overlap on TransactionAlerts

## **Description**

Fixes BlockaidBannerAlert LoadingIndicator overlapping UI and centers icon



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23927?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/23662
Related to: https://github.com/MetaMask/metamask-extension/pull/23773

## **Manual testing steps**

1. Ensure Blockaid is turned on in settings
2. Go to test-dapp
3. Test PPOM buttons 

## **Screenshots/Recordings**



### **Before**

<img width="373" alt="Screenshot 2024-04-10 at 9 47 20 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/ae13e3fa-0c2b-4156-b031-d324e189e556">
<img width="412" alt="320861686-7a2f306d-b4cf-4e3b-b5e0-b0261a69ab5d" src="https://github.com/MetaMask/metamask-extension/assets/20778143/c887fd99-bf19-4b5e-8776-133e15e1311d">


### **After**

<img width="377" alt="Screenshot 2024-04-10 at 9 39 41 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/df9fab2e-68b3-478f-8125-6194213ee95c">
<img width="367" alt="Screenshot 2024-04-10 at 9 39 56 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/36f380b0-d6a4-48c5-b7e1-928bdeadde77">
<img width="373" alt="Screenshot 2024-04-10 at 9 40 07 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/ea5d57c0-e518-43e3-8914-d7fa2c545091">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
